src/lib/dialog/dialog.ts

```diff
@@ -92,11 +92,16 @@ export class MdDialog {
 
   /**
    * Creates the overlay into which the dialog will be loaded.
-   * @param dialogConfig The dialog configuration.
+   * @param config The dialog configuration.
    * @returns A promise resolving to the OverlayRef for the created overlay.
    */
-  private _createOverlay(dialogConfig: MdDialogConfig): OverlayRef {
-    let overlayState = this._getOverlayState(dialogConfig);
+  private _createOverlay(config: MdDialogConfig): OverlayRef {
+    let overlayState = new OverlayState();
+    let strategy = this._overlay.position().global();
+
+    overlayState.hasBackdrop = true;
+    overlayState.positionStrategy = strategy;
```

Comment:
Sorry, I just meant this could still be
```ts
private _getOverlayState(dialogConfig: MdDialogConfig): OverlayState {
  let overlayState = new OverlayState();
  overlayState.hasBackdrop = true;
  overlayState.positionStrategy = this._overlay.position().global();

  return overlayState;
}
```